<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvtkSceneWidget</class>
 <widget class="QWidget" name="QvtkSceneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QvtkSceneWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>QvtkScene</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetQvtkScene"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>QvtkData</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Alias:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAlias"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeViewQvtkData">
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
